web/public/locales/es.json

```diff
@@ -61,7 +61,7 @@
     "dataIsEstimated": "Algunos de los datos de esta región son estimados. Lee más sobre la metodología de estimación <a href=\"https://github.com/electricityMaps/electricitymaps-contrib/wiki/Estimation-methods\" target=\"_blank\">aquí</a>.",
     "exchangesAreMissing": "Los intercambios son considerados en los cálculos pero no son mostrados actualmente. Lee más acerca de nuestros agregados históricos <a href=\"https://github.com/electricityMaps/electricitymaps-contrib/wiki/Historical-aggregates\" target=\"_blank\">aquí</a>.",
     "estimatedTooltip": "Los datos en tiempo real para esta zona son estimados basados en datos de las fuentes y suposiciones del modelo.",
-    "aggregatedTooltip": "Los datos para esta zona son agregados basados en valores horarios históricos",
+    "aggregatedTooltip": "Alternar entre visualización a nivel de zona o vista agregada de país",
```

Comment:
This translation is not correct - but I also don't think this label is used for anything? 

English version is this:
    "aggregatedTooltip": "The data for this zone is aggregated based on historical hourly values",

Reply:
Didn't we use it for this label before?
![image](https://user-images.githubusercontent.com/30777521/220126882-44932202-4444-44ec-baa9-515e55f379d6.png)
(the gray aggregated one)

Reply:
Oh, I've updated the wrong translation 🤦